docs/combine.rst

```diff
@@ -0,0 +1,59 @@
+combine
+=======
+
+.. dfhack-tool::
+    :summary: Combine stacks of food and plants.
+    :tags: fort productivity stockpiles items plants
+
+Usage
+-----
+
+::
+
+    combine <all|here> [--dryrun] [--types=<comma separated list of types>]
```

Comment:
```suggestion
    combine (all|here) [<options>]
```

Reply:
the angle brackets indicate the contents are descriptive. literal text goes in parentheses (or naked like `all|here`